cluster/models.py

```diff
@@ -1,49 +0,0 @@
-import json
```

Comment:
Ah! It was for SQLAlchemy! I thought it was the elastic search one. This looks good once the TODO gets addressed.